website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4544,6 +4546,19 @@ perUser.apply(ParDo.of(new DoFn<KV<String, ValueT>, OutputT>() {
 }));
 {{< /highlight >}}
 
+{{< highlight python >}}
```

Comment:
This Python section does not show up when I have Python selected.

http://apache-beam-website-pull-requests.storage.googleapis.com/13591/documentation/programming-guide/index.html#state-and-timers

Reply:
Done.